### ISSUE TITLE
Should `Argb::as_hex` have a leading pound sign?

### ISSUE THREAD

**Issue by allan2** [Aiving/material-colors#30]
Should `Argb::as_hex` output `#abcdef` or `abcdef`?

Since the rest of the crate has switched to lowercase `abcdef` in usage, should `as_hex` conform?

I ran into it when writing:

*src/scheme/mod.rs tests*
```rs
let c = Argb::from_str("0000ff")?;
let light = SchemeFromPalette::light(c);

assert_eq!(Argb::as_hex(&light.primary), "#343dff");  // pound sign required
```

**Comment by Aiving** [comment]
During the day, I thought about several ways to solve this problem, and here are a few of them:
1. Just remove the pound in `Argb::as_hex`.
2. Add a boolean argument based on which it would be decided whether to add a pound or not.
3. Put it in a separate function like `Argb::as_pound_hex`.
4. Or put it in `fmt::Display` trait (which automatically implements `Argb::to_string`).

**Comment by allan2** [comment]
I feel like `Display` for Argb should output an Argb format.

I like two separate functions, `to_hex` and `to_hex_with_pound`.

I suggest `as_` to `to_` because the returned types are owned, as per [Naming Conventions](https://rust-lang.github.io/api-guidelines/naming.html).

**Comment by Aiving** [comment]
a little bit late, but closed in 808ed5b
